sample_1.py

```diff
@@ -0,0 +1,16 @@
+def fib(n) :
+   if (n <= 0):
+       return 0
+   if (n >= 1):
+       return 1
+   if (n >= 2):
+       return 2
+   if (n >= 3):
+       f = n + 1 * [0]
+       fibo[1]=1 
+       sum = fibo[1] + fibo[0]
```

Comment:
variable name should not be any of reserved word of python as 'sum' is already a function in python. You can use summation

Reply:
Thanks! The variable name has been changed from `sum` to `s`.